test/NuGet.Core.Tests/NuGet.Synchronization.Test/SynchronizationTest.cs

```diff
@@ -182,8 +182,16 @@ public class SynchronizationTests
                 Assert.True(exited, "Timeout waiting for crashing process to exit.");
 
                 // Verify that the process crashed
-                Assert.True(-1 == r1.Process.ExitCode,
-                    $"Failed to self kill, exitcode {r1.Process.ExitCode} {r1.Item2} {r1.Item3}");
+                if (PlatformServices.Default.Runtime.OperatingSystem.Equals("windows", StringComparison.OrdinalIgnoreCase))
+                {
+                    Assert.True(-1 == r1.Process.ExitCode,
+                        $"Failed to self kill, exitcode {r1.Process.ExitCode} {r1.Item2} {r1.Item3}");
+                }
+                else if (PlatformServices.Default.Runtime.OperatingSystem.Equals("linux", StringComparison.OrdinalIgnoreCase))
+                {
+                    Assert.True(137 == r1.Process.ExitCode,
+                        $"Failed to self kill, exitcode {r1.Process.ExitCode} {r1.Item2} {r1.Item3}");
+                }
```

Comment:
what about mac? @davidfowl @victorhurdugaci do you know what is the expected exit code for a killed process?